src/components/MonthRangePicker/MonthRangePickerInput.tsx

```diff
@@ -13,7 +13,7 @@ export function MonthRangePickerInput(props: MonthRangePickerInputProps) {
 
   return (
     <FormControl {...getFormControlProps()}>
-      <BaseMonthRangeInput {...inputProps} {...rest} />
+      <BaseMonthRangeInput invalid={inputProps['aria-invalid']} {...inputProps} {...rest} />
```

Comment:
Does the aria-invalid property need to remain in inputProps, since it has already been used elsewhere?

Reply:
Oh, now i understand hehe